src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -981,6 +983,11 @@ pub struct SharedFsInfo {
 impl SharedFsInfo {
     /// Adjust the configuration information after loading from configuration file.
     pub fn adjust_config(&mut self) -> Result<()> {
+        if self.shared_fs.as_deref() == Some(NO_VIRTIO_FS) {
+            self.shared_fs = None;
+            return Ok(());
+        }
```

Comment:
Just to be super clear, once we return here, it means that no shared_fs option is set. Cool.

Then we don't start virtio-fs daemon, so we don't have to shut the daemon down as well. Cool.
We also need to ensure that when shared_fs = none, we copy the files from host to guest in the case of secrets, service accounts, etc ... and from a quick look this part is missing.  Am I correct here?

Reply:
Thx bro @fidencio , the function of "copy the files from host to guest" has been supported now. Please see [the MR11240](https://github.com/kata-containers/kata-containers/pull/11240)